qubes-rpc/thunar/Makefile

```diff
@@ -0,0 +1,10 @@
+SYSCONFDIR ?= /etc
+LIBDIR ?= /usr/lib
+QUBESLIBDIR = $(LIBDIR)/qubes
+
+.PHONY: install
+
+install:
+	install -D -t $(DESTDIR)$(QUBESLIBDIR) qvm-actions.sh
```

Comment:
You forgot to move this file (see travis).

Reply:
Fix this and 1 more file. I have forgotten to test thunar installation. Tested now.